From ab2965faeeb63fb22b938cc4c80a417b3bf7716b Mon Sep 17 00:00:00 2001
From: Neil Horman <nhorman@gmail.com>
Date: Wed, 7 Jul 2021 08:36:48 -0400
Subject: [PATCH] Fix license specifiers

It was noted that there was an LGPL license in a makefile, which is
rather non-sensical in my mind, and clearly erroneous.  So lets fix up
the other build files to match the spdx header in the source code

Signed-off-by: Neil Horman <nhorman@tuxdriver.com>
---
 Makefile.am     | 19 +------------------
 autogen.sh      |  2 ++
 configure.ac    |  2 ++
 doc/Makefile.am |  1 +
 src/Makefile.am |  3 ++-
 5 files changed, 8 insertions(+), 19 deletions(-)

diff --git a/Makefile.am b/Makefile.am
index fb2cfd6..dba20d4 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -1,24 +1,7 @@
 # Makefile.am --
 # Copyright 2018 Neil Horman
 # All Rights Reserved.
-#
-# This library is free software; you can redistribute it and/or
-# modify it under the terms of the GNU Lesser General Public
-# License as published by the Free Software Foundation; either
-# version 2.1 of the License, or (at your option) any later version.
-#
-# This library is distributed in the hope that it will be useful,
-# but WITHOUT ANY WARRANTY; without even the implied warranty of
-# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
-# Lesser General Public License for more details.
-#
-# You should have received a copy of the GNU Lesser General Public
-# License along with this library; if not, write to the Free Software
-# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
-#
-# Authors:
-#   Steve Grubb <sgrubb@redhat.com>
-# 
+# SPDX-License-Identifier: GPL-2.0-or-later
 
 AUTOMAKE_OPTIONS = no-dependencies
 ACLOCAL_AMFLAGS = -I m4
diff --git a/autogen.sh b/autogen.sh
index b792e8b..35a8644 100755
--- a/autogen.sh
+++ b/autogen.sh
@@ -1,4 +1,6 @@
 #! /bin/sh
+# SPDX-License-Identifier: GPL-2.0-or-later
+
 set -x -e
 mkdir -p m4
 # --no-recursive is available only in recent autoconf versions
diff --git a/configure.ac b/configure.ac
index 3671f27..3b51fb2 100644
--- a/configure.ac
+++ b/configure.ac
@@ -1,3 +1,5 @@
+# SPDX-License-Identifier: GPL-2.0-or-later
+
 AC_INIT(dropwatch,1.5.3)
 AC_PREREQ(2.12)dnl
 AC_CONFIG_HEADERS(config.h)
diff --git a/doc/Makefile.am b/doc/Makefile.am
index 3d61286..76da64e 100644
--- a/doc/Makefile.am
+++ b/doc/Makefile.am
@@ -1,2 +1,3 @@
+# SPDX-License-Identifier: GPL-2.0-or-later
 dist_man_MANS = dropwatch.1 dwdump.1
 
diff --git a/src/Makefile.am b/src/Makefile.am
index f56a39d..1fd6c33 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -1,4 +1,5 @@
-
+# SPDX-License-Identifier: GPL-2.0-or-later
+#
 bin_PROGRAMS = dropwatch dwdump
 
 AM_CFLAGS = -g -Wall -Werror $(LIBNL3_CFLAGS) $(READLINE_CFLAGS)
-- 
2.31.1

